http/vibe/http/common.d

```diff
@@ -619,6 +622,71 @@ FreeListRef!ChunkedOutputStream createChunkedOutputStreamFL(OS)(OS destination_s
 	return FreeListRef!ChunkedOutputStream(interfaceProxy!OutputStream(destination_stream), allocator, true);
 }
 
+/// Parses the cookie from a header field, returning the name of the cookie.
+/// Implements an algorithm equivalent to https://tools.ietf.org/html/rfc6265#section-5.2
+Tuple!(string, Cookie) parseHTTPCookie(string headerString)
```

Comment:
I'd change this to `string parseHTTPCookie(string header_string, scope Cookie dst)`, so that the allocation can be controlled from the outside.